addons/mentions/userstyle.css

```diff
@@ -0,0 +1,12 @@
+.sa-mention {
+  background-color: var(--mentions-color);
```

Comment:
As mentioned above the setting can be removed entirely now that dark-www allows customizing the orange badge colour.
```suggestion
  background-color: var(--darkWww-messageIndicatorColor, #ffab1a);
```

Reply:
This looks pretty bad by default...
![image](https://github.com/user-attachments/assets/af635f17-c9f3-41b8-aa81-f4de45ae14ed)

Reply:
The button color probably works still
![image](https://github.com/user-attachments/assets/e9b26941-d109-41b1-acf6-cf97dd5cd64d)